mllib/src/test/java/org/apache/spark/mllib/regression/JavaRidgeRegressionSuite.java

```diff
@@ -55,7 +56,8 @@ public void runRidgeRegressionUsingConstructor() {
     int numFeatures = 20;
     List<LabeledPoint> data = generateRidgeData(2 * numExamples, numFeatures, 10.0);
 
-    JavaRDD<LabeledPoint> testRDD = jsc.parallelize(data.subList(0, numExamples));
+    JavaRDD<LabeledPoint> testRDD = jsc.parallelize(
+      new ArrayList<LabeledPoint>(data.subList(0, numExamples)));
```

Comment:
Why this change? I don't see how it relates, or why a view of the list would be an issue here.

Reply:
CC @ericl

Reply:
The issue here is that list views are not serializable. This fixes the issue, and does not affect the behaviour of the test.

Reply:
Ah OK. Kind of unrelated, but OK to fix if it came up while working on the issue.